.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Push CI'
+
+on:
+  push:
+
+jobs:
+  test-lint:
+    name: "Lint"
+    secrets: inherit
+    uses: ./.github/workflows/test-lint.yml
```

Comment:
Takto pekne si to vieme skladat. Tu je CI co sa spusta pri kazdom pushe.

Vo workflow pre release napr. nemusime spustat API E2E testy, ked releasujeme CLI.